clientsetup/win/CACreateProjectInitFile.cpp

```diff
@@ -20,8 +20,8 @@
 
 #include "stdafx.h"
 #include "boinccas.h"
-#include "win_util.h"
-#include "base64.h"
+//#include "win_util.h"
```

Comment:
Why keep these commented? Maybe it's better to remove them at all to make code clearer?

Reply:
done